cmd/vpc/db/migrate/public.go

```diff
@@ -3,23 +3,24 @@ package migrate
 import (
 	"github.com/mattes/migrate"
 	"github.com/mattes/migrate/database/postgres"
-	bindata "github.com/mattes/migrate/source/go-bindata"
+	"github.com/mattes/migrate/source/go-bindata"
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog/log"
-	"github.com/sean-/vpc/agent"
-	"github.com/sean-/vpc/config"
+	"github.com/sean-/vpc/cmd/vpc/config"
+	"github.com/sean-/vpc/db"
 	"github.com/sean-/vpc/db/migrations"
 	"github.com/sean-/vpc/internal/buildtime"
 	"github.com/sean-/vpc/internal/command"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-const _CmdName = "migrate"
+const cmdName = "migrate"
```

Comment:
FWIW, just to explain why I do this: I use underscore + capital for signifying an unexported, package-wide constant that isn't a variable but would be if there were an external user of the value.  When reading it's clear it's not a variable on the stack that was passed in as part of the method.  I'm not uptight about it, but I find it to be a more readable style.  `_Foo` indicates something is a `const` and immutable, versus `foo` which reads like a mutable variable.